kolibri/plugins/user/templates/user/user.html

```diff
@@ -1,8 +1,5 @@
 {% extends "kolibri/base.html" %}
-{% load user_tags %}
-{% load kolibri_tags %}
-{% load cache %}
+{% load webpack_tags %}
 {% block content %}
-  {% user_async_assets %}
-  {% user_assets %}
```

Comment:
Need to revert this change for `user_assets` and related hooks, as it is currently being used in the demo_server plugin.